tools/astarte_import/lib/astarte/import.ex

```diff
@@ -148,15 +152,20 @@ defmodule Astarte.Import do
   end
 
   defp xml_event({:startElement, _uri, _l_name, {_prefix, 'stats'}, attributes}, _loc, state) do
-    with total_received_msgs_string = get_attribute(attributes, 'total_received_msgs', "0"),
+    with {:ok, total_received_msgs_string} =
+           get_attribute(attributes, 'total_received_msgs', "0"),
          {total_received_msgs, ""} <- Integer.parse(total_received_msgs_string),
-         total_received_bytes_string = get_attribute(attributes, 'total_received_bytes', "0"),
+         {:ok, total_received_bytes_string} =
+           get_attribute(attributes, 'total_received_bytes', "0"),
          {total_received_bytes, ""} <- Integer.parse(total_received_bytes_string),
-         last_connection_string = get_attribute(attributes, 'last_connection', nil),
-         {:ok, last_connection, 0} <- to_date_or_nil(last_connection_string),
-         last_disconnection_string = get_attribute(attributes, 'last_disconnection', nil),
-         {:ok, last_disconnection, 0} <- to_date_or_nil(last_disconnection_string),
-         last_seen_ip_string = get_attribute(attributes, 'last_seen_ip', nil),
+         {:ok, last_connection_string} =
+           get_attribute(attributes, 'last_connection', "#{DateTime.utc_now()}"),
+         {:ok, last_connection, 0} <- DateTime.from_iso8601(last_connection_string, :extended),
+         {:ok, last_disconnection_string} =
+           get_attribute(attributes, 'last_disconnection', "#{DateTime.utc_now()}"),
+         {:ok, last_disconnection, 0} <-
+           DateTime.from_iso8601(last_disconnection_string, :extended),
+         {:ok, last_seen_ip_string} = get_attribute(attributes, 'last_seen_ip', "0.0.0.0"),
```

Comment:
See above, probably it would be better to keep defaulting these values to `nil` when the information is not there